windows/src/developer/TIKE/dialogs/UfrmMustIncludeDebug.dfm

```diff
@@ -23,19 +19,22 @@ inherited frmMustIncludeDebug: TfrmMustIncludeDebug
     Height = 39
```

Comment:
Just checking the longer caption still fits in the height.

Reply:
Yes, it does 😁